easybuild/easyblocks/c/clang.py

```diff
@@ -407,21 +408,54 @@ def build_with_prev_stage(self, prev_obj, next_obj):
         mkdir(next_obj)
         change_dir(next_obj)
 
-        # Configure.
-        CC = os.path.join(prev_obj, 'bin', 'clang')
-        CXX = os.path.join(prev_obj, 'bin', 'clang++')
+        # Make sure the clang and clang++ compilers from previous stages are (temporarily) in PATH
+        orig_env = copy.deepcopy(os.environ)
+        prev_obj_path = os.path.join(prev_obj, 'bin')
+        current_path = read_environment({'path': 'PATH'})
+        setvar('PATH', current_path['path'] + ":" + prev_obj_path)
```

Comment:
There's no need to add the current location of `clang`/`clang++` to `$PATH` at all.

We know where it is, and we should be stick to passing the full path to the compiler command to `-DCMAKE_*_COMPILER` anyway.

In addition, adding it to the end of `$PATH` isn't wise

Reply:
Done by you in https://github.com/easybuilders/easybuild-easyblocks/pull/2799/commits/21442f8e8dcac3425bb0af18d623a49b5163df18